src/labels/CustomLabels.labels

```diff
@@ -1326,6 +1326,14 @@ If you&apos;re not sure how to resolve these errors, post a message in the Nonpr
         <shortDescription>The total Amount of all Closed Won Opps is not equal to the RD Amount.</shortDescription>
         <value>The combined Amount of all related Closed Won and Pledged Opportunities must match the Amount of a Fixed Length Recurring Donation.</value>
     </labels>
+    <labels>
+        <fullName>RD2_ErrorDataMigrationJobCannotBeRun</fullName>
+        <categories>Recurring-Donations, Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>The job can only be executed when Enhanced RD is enabled in the org</shortDescription>
+        <value>The Enhanced Recurring Donations Data Migration job can only be executed when Enhanced Recurring Donations is enabled in the org.</value>
```

Comment:
@sam-knox Would you mind reviewing this error message? I doubt that a User will ever see this, but it's effectively a fail-safe in case the data migration job is executed when RD2 isn't enabled.